src/menu/rendersettingsmenustate.cpp

```diff
@@ -271,8 +282,11 @@ RenderSettingsMenuState::RenderSettingsMenuState(const std::shared_ptr<MenuRingT
       toggle(engine, engine.getEngineConfig()->renderSettings.doubleUiScale);
     });
 
-  listBox = std::make_shared<CheckListBox>(/* translators: TR charmap encoding */ _("Other"));
+  listBox = std::make_shared<CheckListBox>();
```

Comment:
```suggestion
  listBox = gslu::make_nn_shared<CheckListBox>();
```

Reply:
gslu::make_nn_shared was deleted in [commit 6f5e13b3e6b1355d10db34e813c1129a391fa90b ](https://github.com/stohrendorf/CroftEngine/commit/6f5e13b3e6b1355d10db34e813c1129a391fa90b)
What should I do instead?

Reply:
Ah, sorry, I meant `gsl::make_shared`.